Lab3/server/src/group/controller.ts

```diff
@@ -1,14 +1,35 @@
 import * as express from 'express';
+import * as uuid from 'uuid';
 import { GroupService } from './service';
 
 const app = express();
 
 export default ({ groupService }: { groupService: GroupService }) => {
   // eslint-disable-next-line @typescript-eslint/no-misused-promises
+  app.use(express.json());
+  
   app.get('/', async (_, res) => {
```

Comment:
We need to extend this endpoint or introduce another one to get students by groups